release/nightly_tests/nightly_tests.yaml

```diff
@@ -136,21 +136,21 @@
     compute_template: shuffle/shuffle_compute_large_scale.yaml
 
   run:
-    timeout: 7200
+    timeout: 9000
     prepare: python wait_cluster.py 20 900
     script: python shuffle/shuffle_test.py --num-partitions=5000 --partition-size=200e6
 
 # Stress test for 1TB multi node non-streaming shuffle.
```

Comment:
does it worth to link an issue with it?

Reply:
Do we have an open issue?

Reply:
let's create a P1 for it?

Reply:
I will file an issue separately

Reply:
Btw, I suspect it is highly likely due to pull manager protocol inefficiency.